src/main/mainWindow.ts

```diff
@@ -512,4 +514,5 @@ export async function createWindows() {
     });
 
     initArRPC();
+    if (isWayland) initKeybinds();
```

Comment:
Why are keybinds only enabled for wayland users? I'm an X11 user and keybinds work just fine from my local testing when I remove the conditional.